framer/LayerDraggable.coffee

```diff
@@ -569,12 +572,14 @@ class exports.LayerDraggable extends BaseClass
 		@emit(Events.DragAnimationStart)
 
 	_stopSimulation: =>
+		@emit(Events.Move, @layer.point) if @_isMoving
```

Comment:
idem to above, but here it is the very same property as we are manipulating a little further down ... so here we know exactly what `_isMoving` is supposed to be